src/EA.Weee.Web/Areas/Admin/Controllers/AatfController.cs

```diff
@@ -315,29 +324,31 @@ private async Task<ActionResult> ManageFacilityDetails(FacilityViewModelBase vie
                     var aatf = await client.SendAsync(User.GetAccessToken(), new GetAatfById(viewModel.Id));
```

Comment:
One last thing :) As we have the existingAatf variable we can remove this call to the API

Reply:
Good spot (y)